packages/data/src/onboarding/resolvers.js

```diff
@@ -71,6 +71,10 @@ export function* getTaskLists() {
 	}
 }
 
+export function* getTaskListsByIds() {
+	yield resolveSelect( STORE_NAME, 'getTaskLists' );
```

Comment:
Making use of the `getTaskLists` resolver for now, this does retrieve every task list. This could be changed if we wanted to be more specific.